packages/spruce-types/src/gql/types/Button.gql

```diff
@@ -44,6 +44,9 @@ type Button implements ActionExecutor {
 	"Type attribute for HTML button element. Defaults to 'button'."
 	type: ButtonTypes
 
+	"Otherwise unspecified attributes that will be applied to the underlying button element"
+	HTMLAttributes: JSON
```

Comment:
hey @gcpantazis, why not `htmlAttributes`? I can't recall any places that we use abbreviations, but I do have a memory of seeing `json` in some place and it was lower cased.

Reply:
Actually, lets afterparty on this and decide.